scripts/murmur.ini

```diff
@@ -257,6 +308,15 @@ allowping=true
 ; This option is ignored if Murmur isn't started with root privileges.
 ;uname=
 
+; By default, in log files and in the user status window for privileged users,
+; Mumble will show IP addresses - in some situations you may find this unwanted
+; behavior. If obfuscate is set to true, Murmur will randomize the IP addresses
```

Comment:
obfuscate does not randomize the IP addresses; it SHA-1 hashes them.

Reply:
Then the wiki is wrong as well: https://wiki.mumble.info/wiki/Murmur.ini#obfuscate

However I think for the end-user this detail doesn't really matter, does it?

Reply:
There is a significant difference. Given an IP address you can hash it then check if the hash appears in the log. If it were random, then you would not be able to tell if it appeared in the log given the IP address.

Now that I think about this, the obfuscate feature is flawed since you could just hash all IP addresses and then use the hash in the log to look up the matching IP address.

https://github.com/mumble-voip/mumble/blob/8cdcc28f8f9fc32fed7d48df352eb010398eea62/src/murmur/Server.cpp#L1860-L1876

Reply:
> There is a significant difference. 

True but I still don't think this is a difference the end-user cares much about. In the end what matters is that the IP address is not stored as plain text but obfuscated somehow...
Correcting the wiki might be a good idea nonetheless...

Well I think that's the point though. I think the idea is to still be able to do some checks on the server side whether or not a certain IP exists or is logged or whatever. If this weren't the case, we could simply add an option to not the IP address at all :thinking: 

/cc @davidebeatrici

Reply:
>In the end what matters is that the IP address is not stored as plain text but obfuscated somehow...

My point is that the obfuscation does not work because you can get the IP addresses back in plain text.

>we could simply add an option to not the IP address at all

This seems like a much better idea than what it currently does.

Reply:
Could you please open a new issue for this? :)